src/main/kotlin/no/nav/familie/tilbake/integration/pdl/PdlClient.kt

```diff
@@ -0,0 +1,71 @@
+package no.nav.familie.tilbake.integration.pdl
+
+
+import no.nav.familie.http.client.AbstractRestClient
+import no.nav.familie.http.sts.StsRestClient
+import no.nav.familie.kontrakter.felles.journalpost.Tema
+import no.nav.familie.tilbake.behandling.domain.Fagsystem
+import no.nav.familie.tilbake.common.exceptionhandler.Feil
+import no.nav.familie.tilbake.config.PdlConfig
+import no.nav.familie.tilbake.integration.pdl.internal.PdlHentPersonResponse
+import no.nav.familie.tilbake.integration.pdl.internal.PdlPerson
+import no.nav.familie.tilbake.integration.pdl.internal.PdlPersonRequest
+import no.nav.familie.tilbake.integration.pdl.internal.PdlPersonRequestVariables
+import no.nav.familie.tilbake.integration.pdl.internal.PersonInfo
+import org.springframework.beans.factory.annotation.Qualifier
+import org.springframework.http.HttpHeaders
+import org.springframework.http.HttpStatus
+import org.springframework.stereotype.Service
+import org.springframework.web.client.RestOperations
+import java.time.LocalDate
+
+@Service
+class PdlClient(val pdlConfig: PdlConfig,
+                @Qualifier("sts") val restTemplate: RestOperations,
```

Comment:
Bruk @Qualifier("azure") i stedet og se om du får til å sette opp on-behalf-of  autetisering slik som @hensol sa.

Reply:
blir håndtert i en separat oppgaver https://favro.com/organization/98c34fb974ce445eac854de0/86f241541982404a7491d82d?card=Tea-3562